src/Microsoft.Identity.Web.OWIN/OwinTokenAcquisitionHost.cs

```diff
@@ -55,6 +58,15 @@ public MergedOptions GetOptions(string? authenticationScheme, out string effecti
 
             MergedOptions.UpdateMergedOptionsFromMicrosoftIdentityOptions(_microsoftIdentityOptionsMonitor.Get(effectiveAuthenticationScheme), mergedOptions);
```

Comment:
In the near future, we'll prevent OWIN and SDK to use MicrosoftIdentityOptions (as they are new, and nobody knows about them). For the moment, let's keep this as it's helpful for testing.